File: app/src/main/java/com/constraint/vagabond/data/entities/RecAreaMedia.java
package com.constraint.vagabond.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import androidx.annotation.NonNull;

public final class RecAreaMedia {

  @SerializedName("Title")
  @Expose
  public final String imageTitle;

  @SerializedName("Height")
  @Expose
  public final Integer imageHeight;

  @SerializedName("Width")
  @Expose
  public final Integer imageWidth;

  @SerializedName("URL")
  @Expose
  public final String imageURL;

  public RecAreaMedia(String imageTitle, Integer imageHeight, Integer imageWidth, String imageURL) {

    this.imageTitle = imageTitle;
    this.imageHeight = imageHeight;
    this.imageWidth = imageWidth;
    this.imageURL = imageURL;
  }

  @NonNull
  @Override
  public String toString() {
    String urls = "image_url: " + imageURL +
            "\n" + "image_title: " + imageTitle +
            "\n" + "image_height: " + imageHeight +
            "\n" + "image_width: " + imageWidth;

    return urls;
  }
}

File: app/src/main/java/com/constraint/vagabond/data/entities/RecAreaMedia.kt
package com.constraint.vagabond.data.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class RecAreaMedia(@field:SerializedName("Title")
                   @field:Expose
                   val imageTitle: String, @field:SerializedName("Height")
                   @field:Expose
                   val imageHeight: Int?, @field:SerializedName("Width")
                   @field:Expose
                   val imageWidth: Int?, @field:SerializedName("URL")
                   @field:Expose
                   val imageURL: String)
